Doc/library/tempfile.rst

```diff
@@ -248,6 +248,11 @@ The module defines the following user-callable items:
    The result of this search is cached, see the description of
    :data:`tempdir` below.
 
+   .. versionchanged:: 3.10
```

Comment:
if accepted in 3.9 as a bugfix, i'll update this to 3.9.3 or whatever is appropriate.